src/sidebar/components/NotebookView.js

```diff
@@ -93,6 +104,16 @@ function NotebookView({ loadAnnotationsService }) {
         />
       </div>
       <div className="NotebookView__items">
+        {hasLoadError && (
```

Comment:
The state name `hasLoadError` is quite generic and suggests that _some_ kind of error occurred, but the message is specific to a particular error. Some possible solutions:

1. Make `loadError` an `Error|null` which is set to the actual `Error` if one occurred and then test the type when rendering
2. Make `loadError` an enum or null (eg. `'too-many-results'|'some-other-error'|null`) which specifies the type of error that occurred